### PR TITLE
Handling unknown cv

### PR DESCRIPTION
**Pull request by beaucollins**
See #41

### CODE REVIEW

**Comment by roundhill** [comment]
:shipit:

File: src/simperium/channel.js [hunk 1]
```diff
@@ -403,8 +410,14 @@ Channel.prototype.onChanges = function( data ) {
...
 	// emit ready after all server changes have been applied
 	this.emit( 'ready' );
-}
-;
+};
+
+Channel.prototype.onChangeVersion = function( data ) {
+	if ( data === UNKNOWN_CV ) {
+		this.store.setChangeVersion( null )
+			.then( () => this.startIndexing() );
```
Comment (dmsnell) [inline src/simperium/channel.js]: why create the closure? why not just pass the function?

```js
this.store.setChangeVersion( null ).then( this.startIndexing );
```

Comment (dmsnell) [inline src/simperium/channel.js]: oh. are we dependent upon `this`?

Comment (beaucollins) [inline src/simperium/channel.js]: In this case: yes.

Comment (beaucollins) [inline src/simperium/channel.js]: https://github.com/Simperium/node-simperium/pull/42/files#diff-833f405591f3ebade05af079a2418c0fR349

**Pull request merged**